### PR TITLE
[zh] Localize feature-gates/g*.md

### PR DESCRIPTION
**Pull request by 1000Delta**
Related to issue #44410 

Localize 5 feature-gates/g*.md:
* gce-regional-persistent-disk.md
* generic-ephemeral-volume.md
* graceful-node-shutdown-based-on-pod-priority.md
* graceful-node-shutdown.md
* grpc-container-probe.md

### CODE REVIEW

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/generic-ephemeral-volume.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  render: false
+---
+<!--
+Enables ephemeral, inline volumes that support all features
+of normal volumes (can be provided by third-party storage vendors, storage capacity tracking,
+restore from snapshot, etc.).
+See [Ephemeral Volumes](/docs/concepts/storage/ephemeral-volumes/).
+-->
+启用支持临时的内联卷，这些卷支持普通卷
+（可以由第三方存储供应商提供、存储容量跟踪、从快照还原等等）的所有功能。
```
Comment (windsonsea) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/generic-ephemeral-volume.md:17]:
```suggestion
（可以由第三方存储供应商提供、存储容量跟踪、从快照还原等等）的所有特性。
```
第一个翻的是 特性

**Review by windsonsea — COMMENTED** [review]
/approve

File: content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/generic-ephemeral-volume.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+  render: false
+---
+<!--
+Enables ephemeral, inline volumes that support all features
+of normal volumes (can be provided by third-party storage vendors, storage capacity tracking,
+restore from snapshot, etc.).
+See [Ephemeral Volumes](/docs/concepts/storage/ephemeral-volumes/).
+-->
+启用支持临时的内联卷，这些卷支持普通卷
+（可以由第三方存储供应商提供、存储容量跟踪、从快照还原等等）的所有特性。
```
Comment (tengqm) [inline content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/generic-ephemeral-volume.md:17]:
```suggestion
启用支持临时的内联卷，这些卷支持普通卷的所有特性
（可以由第三方存储供应商提供，支持存储容量跟踪、从快照还原等等）。
```

**Pull request merged**